homeassistant/components/lyric/sensor.py

```diff
@@ -116,6 +132,30 @@ class LyricSensorEntityDescription(
     ),
 ]
 
+ACCESSORY_SENSORS: list[LyricSensorAccessoryEntityDescription] = [
+    LyricSensorAccessoryEntityDescription(
+        key="room_temperature",
+        translation_key="room_temperature",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda _, accessory: accessory.temperature,
+        suitable_fn=lambda _, accessory: accessory.temperature
+        if accessory.type == "IndoorAirSensor"
+        else False,
+    ),
+    LyricSensorAccessoryEntityDescription(
+        key="room_humidity",
+        translation_key="room_humidity",
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=PERCENTAGE,
+        value_fn=lambda room, _: room.roomAvgHumidity,
+        suitable_fn=lambda room, accessory: room.roomAvgHumidity
+        if accessory.type == "IndoorAirSensor"
+        else False,
```

Comment:
```suggestion
        suitable_fn=lambda _, accessory: accessory.type == "IndoorAirSensor",
```

Reply:
Nice catch!  This has been fixed in my most recent commit.